aries_cloudagent/core/conductor.py

```diff
@@ -255,12 +256,19 @@ def inbound_message_router(
         # Note: at this point we could send the message to a shared queue
         # if this pod is too busy to process it
 
-        self.dispatcher.queue_message(
-            message,
-            self.outbound_message_router,
-            self.admin_server and self.admin_server.send_webhook,
-            lambda completed: self.dispatch_complete(message, completed),
-        )
+        try:
+            self.dispatcher.queue_message(
+                message,
+                self.outbound_message_router,
+                self.admin_server and self.admin_server.send_webhook,
+                lambda completed: self.dispatch_complete(message, completed),
+            )
+        except LedgerConfigError:
```

Comment:
Redundancy can be reduced with `except (LedgerConfigError, LedgerTransactionError):`. All the aca-py errors also derive from `BaseError` in case that's useful.